pages/docs/getting-started/product-concepts.mdx

```diff
@@ -76,7 +94,19 @@ See [our Integrations page](/integrations) for an up-to-date list of our support
 
 ## Filter
 
-Filters are a series of rules used to filter data into some desired subset.
+Filters are a series of rules used to filter data into some desired subset. Sometimes you may want to specifically include or exclued rows that meet a criteria. This is where Filters come in.
+
+Each filter rule has a key and operation. The key is a property name, and the operation is a comparison operator.
+
+The following are operations available in Grouparoo's filtering options:
+
+- `equals`
```

Comment:
For these, maybe we want to put the version that would be in the config files as well.  They use a more abbreviated version.  So:
- `eq`: equals
- `ne`: does not equal

They're listed on this page https://www.grouparoo.com/docs/support/config-files what do you think?

Reply:
@tealjulia are substring, notSubstring, and in still used by any plugins? I have checked a few sources.

Reply:
Yep.  BigQuery, Clickhouse, Mongo, MySQL, Postgres, Snowflake, and SQLite.